### ISSUE TITLE
How to manually add a login / credential for an iOS app?

### ISSUE THREAD

**Issue by luckman212** [bitwarden/android#426]
How can I manually add a credential for an iOS app? When I tap "Bitwarden..." to autofill a password, it says "_There are no items in your vault for the app_". Then I click the "+" icon to add one and the URI field is automatically filled out with something like

`
http://localhost:8080/var/containers/Bundle/Application/CAFF654E-097E-4A62-BCEA-55F2E57DBD7E/Domotz%20PRO.app/www/index.html#/welcome`

However, even after saving and then going back to the app to re-attempt login, BW again says "There are no items in your vault for the app". I guess the URI is not correct. How can we detect or determine the right URI?

**Comment by kspearrin** [comment]
This might be due to a bug that was recently fixed and should be available in the upcoming release in a few days. Stay tuned.

**Comment by luckman212** [comment]
I just updated to v1.21.0 that came out today. Still seeing the same behavior. Anything else I need to do, or does this bug need some more digging?

**Comment by kspearrin** [comment]
I am not able to reproduce this issue. Does this happen on all websites or only a certain one?

**Comment by luckman212** [comment]
Hmm you're right. It's definitely not affecting all apps. I guess maybe this is actually a bug with the app itself, that it's not declaring a URI. If I can gather more details I'll update here, otherwise I suppose we can close this.

**Comment by mackaaij** [comment]
I tried to save a password for the iOS app Synology DS File, URI was empty. [Found the Bundle ID](https://pspdfkit.com/guides/ios/current/faq/finding-the-app-bundle-id/) to be com.synology.dsfile but that didn't work as a filter.

Tried to save the password for the iOS app Canva, URL was filled out to https://www.canva.com I think indeed somehow apps should pass the URI but not all apps do. I also tried to manually just add "canva" but that didn't do the job either.

So I'd try adding the website as URI.
